core/domain/user_services.py

```diff
@@ -48,6 +49,8 @@ class UserSettings(object):
         user_id: str. The user id.
         email: str. The user email.
         username: str or None. Identifiable username to display in the UI.
+        role: str. Role of the user. This is used in conjunction with
+            PARENT_ROLES to determine which actions the user can perform.
```

Comment:
role: str. Role of the user used in conjunction with
PARENT_ROLES to determine the actions a user can perform.